src/WebJobs.Extensions.OpenAI/Assistants/AssistantService.cs

```diff
@@ -56,20 +60,46 @@ record InternalChatState(string Id, AssistantStateEntity Metadata, List<ChatMess
 
         this.logger = loggerFactory.CreateLogger<DefaultAssistantService>();
 
-        string connectionStringName = openAiConfigOptions.Value.StorageConnectionName;
+        IConfigurationSection storageConfig = configuration.GetSection(DefaultAssistantService.chatStorage);
```

Comment:
Is this being read from host file - `DefaultAssistantService.chatStorage`, looks like always the constant is being picked?